src/java.base/windows/native/libjava/jni_util_md.c

```diff
@@ -63,34 +63,33 @@ void buildJniFunctionName(const char *sym, const char *cname,
     return;
 }
 
-JNIEXPORT size_t JNICALL
-getLastErrorString(char *buf, size_t len) {
+jstring
+getLastErrorString(JNIEnv *env) {
 
     DWORD errval;
+    WCHAR buf[256];
 
     if ((errval = GetLastError()) != 0) {
         // DOS error
-        size_t n = (size_t)FormatMessage(
+        jsize n = FormatMessageW(
                 FORMAT_MESSAGE_FROM_SYSTEM|FORMAT_MESSAGE_IGNORE_INSERTS,
                 NULL,
                 errval,
                 0,
                 buf,
-                (DWORD)len,
+                sizeof(buf) / sizeof(WCHAR),
```

Comment:
Maybe better to #define the size 256 so that this division is not needed.

Reply:
done